dotnet/TimePlugin/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Should this be in the samples\Demos folder?

Reply:
Sure, good catch.